### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by The-Ladle** [nerve-sparks/iris_android#41] [Q&A] [answered]
I've been paying attention to the project recently and it seems the repo is abandoned. Is this true, or is development simply slow?

**Comment by Dhruvanand24** [comment, marked as answer]
Hi there,

Thank you for your patience. I wanted to clarify that IRIS is not an abandoned project. We work on our projects in dedicated cycles, which allow us to make significant improvements. Iris is indeed being updated soon. While we've been experimenting with other backends, we've encountered limitations in support for models outside of the most popular ones. However, we plan to release updates without changing the backend architecture in the near future.

In addition to backend improvements, we’re also adding a lot of new features and making significant UI changes to enhance user experience. We expect a new release to be available soon, likely within a few days or weeks, depending on the progress.

Stay tuned for the upcoming release, and thank you for your continued support!

---
